assets/components/subscriptionsByCountryGroup/components/digitalSection.jsx

```diff
@@ -30,7 +30,7 @@ type PropTypes = {|
 
 function DigitalSection(props: PropTypes) {
   return (
-    <ThreeSubscriptions heading="">
```

Comment:
out of curiosity, what is the reason this was set as an empty string rather than `null`?